tempesta_fw/http.h

```diff
@@ -259,7 +258,10 @@ typedef struct {
  * Common HTTP message members.
  *
  * @version		- HTTP version (1.0 and 1.1 are only supported);
- * @flags		- message related flags;
+ * @flags		- message related flags. The flags are used in
+ *			  concurrent read and writes, but concurrent writes
+ *			  aren't alowed. So use atomic operations if concurrent
+ *			  updates are possible;
```

Comment:
Atomic bitwise operations work on machine words even though they change just one specific bit. On x86_64 that's 64 bits or `unsigned long`. For that reason it might be safer if the `flags` member is of type `unsigned long`.

By the way, how about moving all flags operations to non-atomic __set_bit/__clear_bit/__flip_bit and the similar atomic set_bit/clear_bit/flip_bit where necessary? Would that be a cleaner solution?